gdrive/core.go

```diff
@@ -195,3 +206,51 @@ func (g *GoogleDriveClient) DownloadToMemory(ctx context.Context, fileID string,
 
 	return data, nil
 }
+
+func generateLargeFile(filename string, size int64) error {
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	data := make([]byte, 10*1024*1024) // 1MB chunk
+	for i := int64(0); i < size/(1024*1024); i++ {
+		_, err := file.Write(data)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (g *GoogleDriveClient) UploadFile(ctx context.Context) (*drive.File, error) {
```

Comment:
is it a test func? maybe we can mark it with comment that it is for testing purposes